src/base/File.h

```diff
@@ -111,10 +111,10 @@ class File
     // Windows-specific HANDLE is needed because LockFileEx() does not take POSIX FDs.
 #if _SQUID_WINDOWS_
     typedef HANDLE Handle;
-    static const Handle InvalidHandle = INVALID_HANDLE_VALUE;
+    static constexpr Handle InvalidHandle = INVALID_HANDLE_VALUE;
```

Comment:
@yadij, the merge bot identifies this PR as a ready-to-be-merged PR with the smallest PR number. If the bot were not running in a staging-only mode, it would have merged this PR. We would like to test the bot reaction to the bot-prepared merge commit (currently https://github.com/squid-cache/squid/commit/9318cb8eb824b71ee775b4b52ee8ee41b20b88b2) being merged out-of-band (i.e., not by the bot itself). I can do that manual merge, but I am not sure whether you are planning to address the above two concerns (PR description improvement and `constexpr` explanation or changes).

* If you are, we will wait for those changes and tell the bot to find another PR.
* Otherwise, I will do that manual merge of the bot-prepared merge commit.

Please let me know either way.